### ISSUE TITLE
install broken

### ISSUE THREAD

**Issue by justinw78** [sandstorm-io/sandstorm#2808]
Hello,
Try to install it with the link  curl https://install.sandstorm.io | bash
after a while it says 
-bash: update-rc.d: not found
and then it stops. 

Can you please help?

greetings justin

**Comment by Michael-S** [comment]
Can you please indicate which version of Linux and which hardware you're running?

**Comment by justinw78** [comment]
yes ofcourse

LSB Version:	:base-4.0-amd64:base-4.0-noarch:core-4.0-amd64:core-4.0-noarch
Distributor ID:	CentOS
Description:	CentOS release 6.8 (Final)
Release:	6.8
Codename:	Final

**Comment by Michael-S** [comment]
Thanks.  Please forgive me for asking the obvious question, but Sandstorm requires kernel 3.13 or newer: https://docs.sandstorm.io/en/latest/install/   - I don't think that's the default kernel for CentOS 6.8, is it?  Could that be the problem?

**Comment by justinw78** [comment]
Hmzz ok i have 
3.10.104-1.el6.elrepo.x86_64
installed

**Comment by Michael-S** [comment]
I don't know if newer kernels are available with CentOS 6.8.  I think the default for CentOS 7.x is still kernel 3.10.  For what it's worth, I ran Sandstorm on Fedora Linux for almost a year with no problems.  ( I only switched Linux distributions away from Fedora because I was having device driver problems with my video card.  It turned out that the problem was the video card, not Fedora, but I never bothered to reinstall Fedora. )

**Comment by justinw78** [comment]
ok Thnx for your time. centos 7 was indeed running at 3.10. Going to find some other software.

greetings.

**Comment by dwrensha** [comment]
As of recently, Sandstorm should work on Linux 3.10: https://sandstorm.io/news/2016-11-10-rhel-centos-arch-support

**Comment by ocdtrekkie** [comment]
There is actually an open PR to amend the docs. https://github.com/sandstorm-io/sandstorm/pull/2802

**Comment by dwrensha** [comment]
@justinw78 it looks like our install script hit trouble for you when it tried to install a sysvinit init script. Your system apparently has an `/etc/init.d` directory but does not have an `update-rc.d`binary, going against an assumption in our install script.

I believe it's possible to take a more manual path through the install script, such that it will ask you whether you want to install an init script.

**Comment by justinw78** [comment]
@dwrensha That would be nice i think:) (manual path )

**Comment by rasos** [comment]
What should be put in the `/etc/init.d` directory? Our install on a Debian jessy also seems to have skipped that part. I only could find in the install script some check for `/etc/sysctl.d/50-sandstorm.conf` which is also not here.

I simply want to achieve that sandstorm is starting at boot. Any hint?

**Comment by kentonv** [comment]
Hi Roland,

If you told the installer that Sandstorm should start as root, then it should have automatically set this up for you. It automatically sets up an initscript or a systemd file depending on what your system uses. If this didn't happen, the installer should have printed a message explaining the problem.

If you avoided granting root privileges (this is uncommon), then you're on your own, since you need to make sure to run `sandstorm start` as the correct user account.

-Kenton

**Comment by rasos** [comment]
Hi Kenton, I installed Sandstorm as root, but it seems that no initscript has been installed automatically, and I did not see a warning message. Now sandstorm does not start at reboot. All the rest is working fine, but I need to start manually. Should I call `sandstorm start` simply in `/etc/rc.local` ?  `service sandstorm restart` does only shutdown sandstorm, not start up again. Can I re-install the initscript with an option?
